test/routes/index.spec.js

```diff
@@ -42,7 +42,7 @@ describe('GET /', () => {
     };
   });
 
-  describe('Homepage', () => {
+  describe.only('Homepage', () => {
```

Comment:
`.only` needs removing

Reply:
Done! Test didn't work, now it does I shall remove.